src/vm/jitinterface.cpp

```diff
@@ -11429,12 +11429,12 @@ void CEEJitInfo::recordRelocation(void * location,
             if (!FitsInRel28(delta))
             {
                 // Use jump stub.
```

Comment:
Since this case is used only for branches, it seemed like `arm64` didn't need the `fAllowRel32` stuff